### ISSUE TITLE
authentication failure is logged at info

### ISSUE THREAD

**Issue by sam-github** [npm/npm#7188]
I keep my log settings at warn, becuase I only want to know about problems.

At warn level, `npm install debug`, for example, hangs silently with no output... at info level, I see:

```
% npm install debug
npm info it worked if it ends with ok
npm info using npm@2.1.6
npm info using node@v0.11.15
npm info attempt registry request try #1 at 12:50:41
npm info retry will retry, error on last attempt: Error: This request requires auth credentials. Run `npm login` and repeat the request.
npm info attempt registry request try #2 at 12:50:51
npm info retry will retry, error on last attempt: Error: This request requires auth credentials. Run `npm login` and repeat the request.
```

**Comment by smikes** [comment]
This is a duplicate of #6644, fixed in `npm@2.1.9` and later versions by commit <https://github.com/npm/npm/commit/9bb2c3435cedef40b45d3e9bd7a8edfb8cbe7209>

**Comment by sam-github** [comment]
I'm puzzled as to why the 0.11.15, released yesterday, doesn't include an npm2 fix from last November. Thanks for the update.
